### ISSUE TITLE
Transition.Change not visible inside FlatList with multiple columns

### ISSUE THREAD

**Issue by emeraldsanto** [software-mansion/react-native-reanimated#697]
I'm currently implementing a screen where a user can manage their subscriptions, they are displayed inside a `FlatList` (imported from React Native) with `numColumns` set to `2`. The transition is defined as follows:

```js
<Transition.Sequence>
    <Transition.Out type="fade" durationMs={100} />    
    <Transition.Change durationMs={100} />
</Transition.Sequence>
```

When I remove an item from my state, the changes reflect but the `fade` seems to apply to all items and the `change` transition is not visible. This all goes away when I remove them `numColumns` prop. This may not be a bug in this library but rather related to how a multi-column `FlatList` behaves, it looks like it is re-rendering entirely instead of simply removing the selected item.

I'm using `react-native-reanimated` version `1.7.1`.

**Comment by jkadamczyk** [comment]
Can you provide a repro in form of Snack that we can immediately run?

**Comment by jakub-gonet** [comment]
We discussed Transitions internally in our team. Considering that v2 makes creating animations easier and Transitions was an experimental API for a long time, we decided to leave Transitions as they are.
 
We won’t actively fix bugs in them, but we’ll happily merge any PR with fixes. We’re also thinking about rewriting Transitions in v2 sometime in the future.

I’m closing this issue, but if you have further questions, I will be happy to help.
